untidy/contaminators.py

```diff
@@ -1,9 +1,109 @@
 """ Functions to contaminate the data """
+import pandas as pd
+import numpy as np
+import random
 
 """ Column-wise contaminators """
 
+def data_check(data):
+    """
+    Check input data
+
+    Params
+    ------
+    data: pd.Series, pd.DataFrame
+        data to check
+
+    Returns
+    -------
+    row_idxs, col_idxs: int(s)
+        integers specifying, respectively, the amount of rows and columns in data.
+        These values will be passed to *random_indeces* in order to generate the indeces to be contaminated
+
+    """
+    if isinstance(data, pd.Series):
+        row_idxs = data.shape[0]
+        col_idxs = 2
+
+    elif isinstance(data, pd.DataFrame):
+        row_idxs, col_idxs = data.shape 
+        
+    else:
+        raise TypeError('data should be pd.Series or pd.DataFrame')
+
+    return row_idxs, col_idxs
+
+def random_indices(row_idxs, col_idxs, corruption_level = 4):
+    """
+    Get random indeces to contaminate
+
+    Parameters
+    ----------
+    row_idxs: int
```

Comment:
Could we rename `row_idxs` to num_rows and col_idxs to `num_cols` or something less confusing everywhere in the code?